aries_cloudagent/admin/server.py

```diff
@@ -351,6 +364,8 @@ async def check_multitenant_authorization(request: web.Request, handler):
 
                 is_multitenancy_path = path.startswith("/multitenancy")
                 is_server_path = path in self.server_paths or path == "/features"
+                # allow base wallets to trigger update through api
+                is_upgrade_path = path.startswith("/anoncreds/wallet/upgrade")
```

Comment:
This allows the base wallet to upgrade when the tenant authorization header is null. You'd need to add future upgrade paths if you wanted to allow the base wallet to trigger an upgrade through the api.

It might possibly have some conflict with your authorization decorators PR.